Fix feedback error

krischarbonneau opened the pull request:
# Description

[Feedback api returning an HTTP 400 error](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-558)

Fixed an issue where the feedback widget was returning a HTTP 400 error when trying to submit feedback. The Feedback API requires that the project name and url be included in the payload alongside the feedback, but only the feedback was being included, resulting in the 400 error.

I created a new reference object with all the required fields and pass the project name and path to the `Layout` component (to be passed to `PhaseBanner` and then the `FeedbackWidget`). 

## Acceptance Criteria

When a user types feedback into the feedback form and clicks submit, they are shown a green banner confirming that their feedback has been sent and an email is sent to the proper inbox with this feedback.

## Test Instructions

1. Pull in branch
2. Change `NEXT_PUBLIC_NOTIFY_REPORT_A_PROBLEM_EMAIL`'s value in your `.env.local` to your own email
3. Type `npm run dev`
4. Go to `/projects/digital-centre` and click `Give feedback to help improve this website`
5. Enter feedback into the form field and click submit
6. Check your email for the feedback


## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)
- [x] Unit tests have been added/updated
- [x] Update CHANGELOG

## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)

var-kyle reviewed (changes requested):
Using the top link to open the feedback works great, but using the button at the bottom is throwing an error.

![image](https://user-images.githubusercontent.com/34345435/144918830-b4a60474-9f4f-4ce4-b201-408722080d74.png)

krischarbonneau commented:
> Using the top link to open the feedback works great, but using the button at the bottom is throwing an error.
> 
> ![image](https://user-images.githubusercontent.com/34345435/144918830-b4a60474-9f4f-4ce4-b201-408722080d74.png)

This was a tricky one. Added back functionality for the bottom button, and also changed `ActionButton` to include the `ariaExpanded` prop so the aria-expanded state can be manipulated whenever the bottom feedback window is opened or closed (since it is manipulated through the `CallToAction` component).

I also removed the srSpeak function from `PhaseBanner` as it doesn't appear that it was even needed since it was double announcing the toggle state.

var-kyle reviewed (approved):
Nice, it's working as expected now.
